docs/intro-to-lbp-schema.md

```diff
@@ -0,0 +1,42 @@
+
+Further, the LombardPress-Schema is not only concerned with facilitating the ease of presentation of an individual text or text fragment, but is also concerned to help facilitate, through the adoption of the ontology and guidelines of the Scholastic Commentaries and Texts Archive, the construction of a network corpus from previously isolated text fragments.
+
+# Relationship to the Scholastic Commentaries and Texts Archive
+
+While the LombardPress-Schema has been conceived and built alongside the development of the [Scholastic Commentaries and Texts Archive (SCTA)](http://scta.info), it is important that these two components are understood as distinct and modular parts of a larger ecosystem.
+
+The *Scholastic Commentaries and Texts Archive* is, at its core, an ontology and schema for a set of ideas that, taken together, constitute a text-network. Manuscripts, books, images, and digital transcriptions (as well as authors, editors, references, quotations, and citations) occupy distinct places within this vast network. The SCTA ontology aims to organize all of these distinct but related resources into a connected whole. As part of this process, it offers a series of resource ids that editors can make use of when editing to add more information to this network. In turn the SCTA build script can crawl semantically encoded editions to extract required metadata. However, in order to perform this aggregation, the SCTA crawler must be able to understand ahead of time the semantic structure of the data it is crawling. Such understanding requires that data be prepared according to a known standard.
+
+The LombardPress-Schema is one such known standard. The LombardPress-Schema pays close attention to the ontology, resources ids, and general recommendations of the SCTA and offers explicit guidelines about how these resources ids should be employed in the editing process. But the LombardPress-Schema need not be the only standard that the SCTA build script can accommodate. It is quite possible that someday a subdomain of scholastic texts will demand a schema that the LombardPress-Schema cannot accommodate. This new schema can make its own provisions for how to use the SCTA ontology to connect to the wider corpus. The SCTA build script can be then be modified to crawl two distinct, but standardized and documented, data serializations. Nevertheless, the LombardPress-Schema is, at the present, the most mature attempt both to provide a detailed encoding standard for all scholastic texts and to offer guidelines about how to prepare a text in such a way that it can be easily absorbed within the *Scholastic Commentaries and Texts Archive*.
+
+# Status and Future Development
```

Comment:
This is a great section. I really like making the best practice principles explicit. But I'm in doubt if the heading reflects the content. Based on the headline I would expect a development roadmap or something similar. Should the heading be changed? "Development principles and best practices" or something?